generic_config_updater/change_applier.py

```diff
@@ -72,6 +72,10 @@ def remove_backend_tables_from_config(self, data):
 
 class ChangeApplier:
 
+    # Current Service Validator is global wide validating and restarting,
+    # if any of ASIC has run it, no need run again.
+    has_run_service_validator = False
```

Comment:
Hi @xincunli-sonic , I don't need this part from my understanding.
Current service validator will not impact asic's service, since the service related table is not present in ascis.
However, if we do have service need restart with asics, we still need to run the service validator based on patch.
If a patch contains table related to service validator, we still need to run validator rather than only run it once in host.

Reply:
Make sense, remove this commit.